service/src/main/java/com/codeforcommunity/processor/authenticated/ProtectedUserProcessorImpl.java

```diff
@@ -180,4 +193,44 @@ public UserListResponse getAllUsers(JWTData userData, Country country) {
 
     return new UserListResponse(response);
   }
+
+  @Override
+  public UserListResponse getDisabledUsers(JWTData userData, Country country) {
+
+    if (!userData.isAdmin()) {
+      throw new AdminOnlyRouteException();
+    }
+
+    List<UserDataResponse> response = new ArrayList<>();
+    List<UsersRecord> users = new ArrayList<>();
+
+    if (country == null) {
+      users =
+          db.selectFrom(USERS)
+              .where(USERS.DELETED_AT.isNull())
+              .and(USERS.DISABLED.eq(true))
+              .fetch();
+    } else {
+      users =
+          db.selectFrom(USERS)
+              .where(USERS.COUNTRY.eq(country))
+              .and(USERS.DELETED_AT.isNull())
+              .and(USERS.DISABLED.eq(true))
+              .fetch();
+    }
+
+    for (UsersRecord user : users) {
```

Comment:
Would prefer to map with stream syntax or fetchInto a list of Users instead of using a for loop